eng/Versions.props

```diff
@@ -43,7 +43,7 @@
       src\Components\WebAssembly\Sdk\testassets\Directory.Build.props, and
       src\Razor\test\testassets\Directory.Build.props when this changes.
     -->
-    <DefaultNetCoreTargetFramework>net6.0</DefaultNetCoreTargetFramework>
+    <DefaultNetCoreTargetFramework>net7.0</DefaultNetCoreTargetFramework>
```

Comment:
/btw it looks like we need to change `$(DefaultNetFxTargetFramework)` to `net462` at the same time. Were the target TFMs for System.IO.Pipelines and System.Runtime.CompilerServices.Unsafe changed recently @ericstj @ViktorHofer❔

Reply:
Yes - see https://github.com/dotnet/runtime/pull/58011

Reply:
Yes, .NET 7 packages support net462 and above but not net461: https://devblogs.microsoft.com/dotnet/net-framework-4-5-2-4-6-4-6-1-will-reach-end-of-support-on-april-26-2022/

Reply:
Ah, I see. Sounds like we should update here too, then.